handbook/information-security/supplier-policy.md

```diff
@@ -0,0 +1,115 @@
+# Supplier Policy
+
+In providing our services we rely on a number of third party suppliers that have
+access to some of the sensitive information we handle. This policy sets out how
+we vet our suppliers to ensure that we understand and assess the risks involved.
+
+## Objectives
+
+The objectives of our Supplier Policy are to:
+* understand which of our suppliers are handling sensitive information on our
+  behalf
+* understand the risks created by our collaboration
+* ensure the relevant suppliers follow best practices to minimise information
+  security risks
+* work with our suppliers to implement best practice where it is not being
+  implemented
+* make a conscious decision whether the risks we expose ourselves to are
+  acceptable
+* where we are exposed to unacceptable risk, mitigate this by ending the
+  collaboration if necessary
+
+## Vetting Process
+The below diagram illustrates the process that the rest of this document
+describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+Our [Supplier
+List](https://docs.google.com/spreadsheets/d/14kog_nFut3v2t5mZNNatZ-oWb7tP3ImWdXjUP9EkZGc/edit?usp=sharing)
```

Comment:
Not sure I would have this link here. It's not publicly available (or shouldn't be) and we can just reference that the doc exists.